README.md

```diff
@@ -220,10 +289,6 @@ and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
   current moment, having the file handled by the `template` module now makes adding that functionality much
   simpler than if the file is initially handled by the `copy` and then needs to be moved before it can be
   edited.
-* When using the `template` module, refrain from appending `.j2` to the file name. This alters the syntax
```

Comment:
This entry has justification for why it is so. You propose to remove it and replace it without justification as to why. Can you explain your reasoning?

Reply:
Fixed. Put it back since this is an ongoing discussion.